src/components/chat/w2wChat/chatBox/ChatBox.tsx

```diff
@@ -15,7 +15,7 @@ import ScrollToBottom from 'react-scroll-to-bottom';
 import { useClickAway } from 'react-use';
 import styled, { useTheme } from 'styled-components';
 import { produce } from 'immer';
-import { ChatProfile, ChatViewList, MessageInput } from '@pushprotocol/uiweb';
+import { ChatProfile, ChatViewList, MessageInput, ChatUIProvider, darkChatTheme, lightChatTheme } from '@pushprotocol/uiweb';
```

Comment:
we already have chatuiprovider in chatmodule so why is it added here as well ?

Reply:
@pritipsingh add it over here instead
https://github.com/ethereum-push-notification-service/push-dapp/blob/4eeebf14e17150770883e66217ca0bfc80ff038a/src/modules/chat/ChatModule.tsx#L351